src/manage_nsfs/health.js

```diff
@@ -66,6 +66,10 @@ const health_errors = {
         error_code: 'INVALID_CONFIG_DIR',
         error_message: 'Config directory is invalid',
     },
+    LOW_STORAGE: {
+        error_code: 'LOW_STORAGE',
+        error_message: 'FS storage is low.'
```

Comment:
If this error code only used for notification log file you can add more details like `FS storage is low for notification log directory`